script1

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
```

Comment:
Write a comment here saying what this script does and how to run it. It would be nice if you could write some examples. For example:
```
Example Usage:
./script1 a1 a2 a3
n=3

Output::
The 2nd parameter is a1
The (n=3)th parameter is a3
```

I think that your code does not display the correct value (it doesn't check if the user entered the 2nd parameter